### PR TITLE
Add clang version to Info.plist

### PR DESCRIPTION
**Pull request by dnfield**
If we have an engine built with bitcode, we'll want to verify in the framework (e.g. flutter doctor) whether we're using the same version of Xcode to build the IPA.  This will allow for that.

In an ideal world, we'd be passing in the clang++ path as an argument, but I just can't find a good way to do that.  This duplicates logic in our buildroot to figure out the path to clang based on whether bitcode is enabled.

Part of the effort for https://github.com/flutter/flutter/issues/15288

### CODE REVIEW

File: build/copy_info_plist.py [hunk 1]
```diff
@@ -25,7 +26,12 @@ def main():
   text = open(sys.argv[1]).read()
   engine_path = os.path.join(os.getcwd(), "..", "..", "flutter")
   revision = git_revision.GetRepositoryVersion(engine_path)
-  text = text.format(revision)
+  clang_executable = str(os.path.join("..", "..", "buildtools", "mac-x64", "clang", "bin", "clang++"))
+  if sys.argv[3] == "--bitcode=true":
```
Comment (gaaclarke) [inline build/copy_info_plist.py]: Please update the usage comment at the top of the file.

File: build/copy_info_plist.py [hunk 2]
```diff
@@ -25,7 +26,12 @@ def main():
...
+    clang_executable = "clang++"
+  version = subprocess.check_output([clang_executable, "--version"])
```
[the remark this answers is not in the document]
Comment (dnfield) [reply build/copy_info_plist.py]: Done

**Comment by gaaclarke** [comment]
There are 2 build files that call this script, you've only updated one.  One sec let me grab the other one.

**Comment by gaaclarke** [comment]
The way you've written the 3rd commandline argument you'll need to edit this too: https://github.com/flutter/engine/blob/8849d3092d5aa6866501663cd62397f1a1564e67/shell/platform/embedder/BUILD.gn#L156

**Comment by dnfield** [comment]
Does that look right?

**Pull request merged**